.cbmc-batch/source/openssl/evp_override.c

```diff
@@ -0,0 +1,472 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include <openssl/evp.h>
+
+#include <ec_utils.h>
+#include <make_common_data_structures.h>
+#include <proof_helpers/nondet.h>
+
+/* Abstraction of the EVP_PKEY struct */
+struct evp_pkey_st {
+    int references;
+    EC_KEY *ec_key;
+};
+
+/*
+ * Description: The EVP_PKEY_new() function allocates an empty EVP_PKEY structure which is used by OpenSSL to store
+ * public and private keys. The reference count is set to 1. Return values: EVP_PKEY_new() returns either the newly
+ * allocated EVP_PKEY structure or NULL if an error occurred.
+ */
+EVP_PKEY *EVP_PKEY_new() {
+    EVP_PKEY *pkey = can_fail_malloc(sizeof(EVP_PKEY));
+
+    if (pkey) {
+        pkey->references = 1;
+        pkey->ec_key     = NULL;
+    }
+
+    return pkey;
+}
+
+/*
+ * Description: EVP_PKEY_get0_EC_KEY() also returns the referenced key in pkey or NULL if the key is not of the correct
+ * type but the reference count of the returned key is not incremented and so must not be freed up after use. Return
+ * value: EVP_PKEY_get0_EC_KEY() returns the referenced key or NULL if an error occurred.
+ */
+EC_KEY *EVP_PKEY_get0_EC_KEY(EVP_PKEY *pkey) {
+  assert(pkey);
```

Comment:
Inconsistent indentation throughout this file.

Reply:
Should be fixed now.